app/views/admin/theses/show.html.erb

```diff
@@ -30,6 +30,13 @@ as well as a link to its edit page.
       [:edit, namespace, page.resource],
       class: "button",
     ) if accessible_action?(page.resource, :edit) %>
+    <%= link_to(
```

Comment:
I have some concern about adding this button back in. I suspect there's a reason it's not there in the first place. That said, I trust Wilder not to delete anything that shouldn't be deleted, so if you're comfortable with it, I think it's fine. (At the very least, it will make future upgrades less confusing.)

Reply:
I couldn't actually find where it was in our code ever and assumed it was new to the template honestly.

As it's conditional as to whether it displays, maybe we check in with Wilder as to whether they can see this button... and if they can see it, adjust permissions until they can't?

Reply:
Yeah, I think you're right. I see the destroy button in that view for the [v0.19 release commit](https://github.com/thoughtbot/administrate/blob/482bc010581440d289074ff454c85cff4e65c386/app/views/administrate/application/show.html.erb#L33-L40), but _not_ in some earlier releases. It seems they added it at some point, and we just didn't pull it in to the customized template. Knowing that it likely wasn't an intentional customization makes me feel better about keeping it.

Incidentally, I confirmed that it's visible to the `thesis_admin` role, so Wilder should be able to see it.